rust/psibase/src/services/tokens/memo.rs

```diff
@@ -54,7 +54,6 @@ impl<'a> Unpack<'a> for Memo {
             .try_into()
             .map_err(|_| fracpack::Error::ExtraData)?;
```

Comment:
The cast isn't needed. Comparison as a usize is fine. I would try to make this as similar to `new` as possible, to make it more obvious that the checks are equivalent.